css/styles.css

```diff
@@ -0,0 +1,194 @@
+* {
+  font-family: "Just Me Again Down Here", cursive;
+}
+
+body {
+  background-image: url("https://images.ctfassets.net/usf1vwtuqyxm/71J6YPxNauhFABbjHpGluJ/631bd6980b0a430d5b476ef331abb2b5/release_1_hogwarts.png");
+  background-repeat: no-repeat;
+  background-attachment: fixed;
+  background-size: cover;
+}
+
+.nav-bar {
+  display: flex;
+  margin: 0 auto;
+  color: #f89b9b;
+  font-size: 20px;
+}
+
+.column-one-fourth {
+  position: absolute;
+  top: 0;
+  right: 0;
+}
+
+.column-one-fourth-left {
+  position: absolute;
+  top: 0;
+  left: 0;
+}
```

Comment:
You shouldn't be using absolute positioning on your columns or elements related to your layout